### ISSUE TITLE
geometry.remove.min-length leads to different speed limits in different directions

### ISSUE THREAD

**Issue by ambuehll** [eclipse-sumo/sumo#10539]
`geometry.remove.min-length xx` can lead to different speed limits (as an example) on opposite lanes. 
It seems that it merges in the direction of travel, which is fine, but maybe when the road is bi-directional, both sides should match. Maybe the longer can overrule the smaller segment, when joining?

geometry.remove.min-length 50
<img width="713" alt="image" src="https://user-images.githubusercontent.com/48930860/162961857-00b7774d-04df-41d3-9b8a-a023da5916c3.png">

vs
geometry.remove.min-length not active

<img width="472" alt="image" src="https://user-images.githubusercontent.com/48930860/162961925-cbb7000f-ad78-4445-8403-3bdf60be4548.png">

[files](https://drive.google.com/drive/folders/1o32n1Cki6nW0K0PHvo9qjTWdBjhMLe9e?usp=sharing)
**SUMO-version:**current

Thanks!

Lukas

**Comment by behrisch** [comment]
Yes it seems currently always the first segment wins and "majority vote" sounds like a good idea to me. Probably this should apply to all properties and even to the id then.

**Comment by gladysmcgan** [comment]
Hi, I would like to bring this up again, sorry to latch on an old issue! This is still a case which results in short edges, which affects the simulation sometimes, for example, when cars are unable to change lanes in time and end up teleporting after waiting (unfortunately I cannot reproduce this on a small network..)

![Image](https://github.com/user-attachments/assets/ab5a0848-3bc8-4251-a802-1abcc99ad9cd)

The edge after geometry.remove.min-length would take the properties first edge (no bus lane), when the ideal would be that the whole edge has the bus lane. 

A fix for this would be a great help :)

**Comment by namdre** [comment]
No need to replicate the simulation behavior. It's enough if you can provide a small input for netconvert and specify the expected result of using the option.

**Comment by gladysmcgan** [comment]
[geometry_remove_dist.zip](https://github.com/user-attachments/files/19585709/geometry_remove_dist.zip)

The expected output is that edge -1354339964 is joined with edge -1354339966, but edge -1354339966 defines the properties of the resulting combined edge (i.e. bus lane going up to the junction 1474909969)
